lib/plugins/rest.js

```diff
@@ -40,7 +40,7 @@ exports.register = function (server, options, next) {
 			instances.createInstance(ami, type)
 			.then(function (instance) {
 				result = instance;
-				return awsAdapter.runInstances(instance);
+				return awsAdapter.runInstances(instance, instances, server.log.bind(server));
```

Comment:
Same for the log function. I think these artifacts may end up getting used across several methods.